dspace-api/src/main/java/org/dspace/core/AbstractHibernateDAO.java

```diff
@@ -105,127 +106,87 @@ public T findByID(Context context, Class clazz, int id) throws SQLException {
      */
     public List<T> findMany(Context context, Query query) throws SQLException {
         @SuppressWarnings("unchecked")
-        List<T> result = (List<T>) query.list();
+        List<T> result = (List<T>) query.getResultList();
         return result;
     }
 
-    /**
-     * Create criteria matching an entity type or a supertype thereof.
-     * Use when building a criteria query.
-     *
-     * @param context         current DSpace context.
-     * @param persistentClass specifies the type to be matched by the criteria.
-     * @return criteria concerning the type to be found.
-     * @throws SQLException passed through.
-     */
-    public Criteria createCriteria(Context context, Class<T> persistentClass) throws SQLException {
-        return getHibernateSession(context).createCriteria(persistentClass);
-    }
-
-    /**
-     * Create criteria matching an entity type or a supertype thereof.
-     * Use when building a criteria query.
-     *
-     * @param context         current DSpace context.
-     * @param persistentClass specifies the type to be matched by the criteria.
-     * @param alias           alias for the type.
-     * @return criteria concerning the type to be found.
-     * @throws SQLException passed through.
-     */
-    public Criteria createCriteria(Context context, Class<T> persistentClass, String alias) throws SQLException {
-        return getHibernateSession(context).createCriteria(persistentClass, alias);
-    }
+//    public Criteria createCriteria(Context context, Class<T> persistentClass) throws SQLException {
+//        return getHibernateSession(context).createCriteria(persistentClass);
+//    }
+//
+//    public Criteria createCriteria(Context context, Class<T> persistentClass, String alias) throws SQLException {
+//        return getHibernateSession(context).createCriteria(persistentClass, alias);
+//    }
 
-    /**
-     * Create a parsed query from a query expression.
-     *
-     * @param context current DSpace context.
-     * @param query   textual form of the query.
-     * @return parsed form of the query.
-     * @throws SQLException
-     */
     public Query createQuery(Context context, String query) throws SQLException {
         return getHibernateSession(context).createQuery(query);
     }
 
-    /**
-     * Get the entities matched by the given Criteria.
-     * Use this if you need all results together.
-     *
-     * @param criteria description of desired entities.
-     * @return the entities matched.
-     */
-    public List<T> list(Criteria criteria) {
-        @SuppressWarnings("unchecked")
-        List<T> result = (List<T>) criteria.list();
-        return result;
-    }
-
-    /**
-     * Get the entities matching a given parsed query.
-     * Use this if you need all results together.
-     *
-     * @param query the query to be executed.
-     * @return entities matching the query.
-     */
-    public List<T> list(Query query) {
```

Comment:
And another method comment removed.

Reply:
Can we just recheck these classes and ensure *all public methods* have JavaDocs?  In general, that's what we should strive for, and I currently see a lot of public methods where comments were either removed, or new methods were added with no comments

Reply:
The removal of documentation was entirely accidental and I will re-add them and write documentation for new methods as soon as possible.